source/partials/next_release/_release.md.erb

```diff
@@ -1,73 +1,49 @@
 
-  Page headers refreshed across application
-  <figure class='shrink-figure-50'>
-      <%= image_tag 'releases/17.11.0/vsm-explicit-link-stage-details.png', alt: 'Explicit vsm link in stage details header' %>
-  </figure>
+     With this change, an authenticated user is re-directed to home page on trying to access the login page.
+* <%= link_to_commit '6fda2f5cb5104b9394f2cfd5e41ca1e517078bd6', 'Add new unlock behavior', 3943%> <%= link_to_issue 106 %>
 
-  <figure class='shrink-figure-50'>
-      <%= image_tag 'releases/17.11.0/vsm-explicit-link-vsm-header.png', alt: 'Explicit vsm link in vsm header' %>
-  </figure>
+    As a part of this change, two types of locking behavior are introduced:
 
-  <figure class='shrink-figure-50'>
-      <%= image_tag 'releases/17.11.0/vsm-explicit-link-job-details-header.png', alt: 'Explicit vsm link in job details page header' %>
-  </figure>
+    * Run single instance of pipeline at a time ("Lock behavior: Unlock when finished")
 
-<h4>Other Improvements</h4>
-
-* <%= link_to_commit 'eb7305e41c03df0266974c6b7f432aac4f491cf1', 'Improves classloading and resource management', 3792 %>
+    * Run single instance of pipeline and lock on failure ("Lock behavior: Lock on failure")
 
-  Prevents file-descriptor and classloader leaks by ensuring that:
+    More information can be found in the <%= link_to("docs", "https://docs.gocd.org/17.12.0/configuration/admin_lock_pipelines.html") %>
 
-  * Classloaders for agent-launcher and agent are closed properly
-  * Temporary jar files that are extracted are removed after closing the
-    classloaders
-  * File watcher from log4j is replaced with a custom implementation
+<h4>Bug Fixes</h4>
+* <%= link_to_commit 'ab1ccf83aa90dfa3fc916ca892592e6db81717fc', 'Update server side validation to not use file system', 4026%> <%= link_to_issue 3984 %>
+* <%= link_to_commit 'e959f9d436e6c01ae61d080bd37b68117c7076d0', 'Fix multiple logs showing on build details page\'s console tab', 4024%> <%= link_to_issue 4018 %>
+* <%= link_to_commit '6660a1b093d1c265f697c6e0ee16d3c146772a24', 'Show console log not available error in build details page\'s console tab when console.log file is not available', 3998%> <%= link_to_issue 3773 %> <%= link_to_issue 3650 %>
+* <%= link_to_commit 'e48d9588988969c42c2f63b617cd9d08f8ca015d', 'Fixes console.log file not being available at server if websocket connection is closed while waiting for logs', 3999%> <%= link_to_issue 3986 %>
+* <%= link_to_commit 'ee992e6175497a3ae927c9e956e22d822e5715bd', 'Fixes issue in agent zip installers due to which agents fail to register if their log directories\' path have spaces in them', 3976%> <%= link_to_issue 3966 %>
+* <%= link_to_commit 'e6c9b3ad5def1b6ea682c6723106f8c77df94f92', 'Fixes issue with CCtray in which pipelines visible to users belonging to particular plugin role, would\'nt be listed in it.', 3955%> <%= link_to_issue 3954 %>
```

Comment:
May be simplify it to say `Fixes issue with stale cache being used in CCtray which caused users associated with plugin-roles to not see pipelines which they were authorized to see otherwise`